### ISSUE TITLE
Can't dump QFIL image properly

### ISSUE THREAD

**Issue by NoaHimesaka1873** [AndroidDumps/Firmware_extractor#42]
```
Create Temp and out dir
grep: warning: stray \ before /
Extracting firmware on: /home/yuifunami/dumpyara/working/Archive
QFIL detected
flag needs an argument: -x
Usage of /home/yuifunami/dumpyara/Firmware_extractor/tools/Linux/bin/packsparseimg:
  -o string
        output dir path (default "./")
  -t string
        image to unsparse: system / cache / userdata / persist / vendor (default "system")
  -x string
        xml file to load (default "./rawprogram_unsparse.xml")
mv: cannot stat 'system.raw': No such file or directory
flag needs an argument: -x
Usage of /home/yuifunami/dumpyara/Firmware_extractor/tools/Linux/bin/packsparseimg:
  -o string
        output dir path (default "./")
  -t string
        image to unsparse: system / cache / userdata / persist / vendor (default "system")
  -x string
        xml file to load (default "./rawprogram_unsparse.xml")
mv: cannot stat 'vendor.raw': No such file or directory
flag needs an argument: -x
Usage of /home/yuifunami/dumpyara/Firmware_extractor/tools/Linux/bin/packsparseimg:
  -o string
        output dir path (default "./")
  -t string
        image to unsparse: system / cache / userdata / persist / vendor (default "system")
  -x string
        xml file to load (default "./rawprogram_unsparse.xml")
mv: cannot stat 'modem.raw': No such file or directory
flag needs an argument: -x
Usage of /home/yuifunami/dumpyara/Firmware_extractor/tools/Linux/bin/packsparseimg:
  -o string
        output dir path (default "./")
  -t string
        image to unsparse: system / cache / userdata / persist / vendor (default "system")
  -x string
        xml file to load (default "./rawprogram_unsparse.xml")
mv: cannot stat 'dtbo.raw': No such file or directory
flag needs an argument: -x
Usage of /home/yuifunami/dumpyara/Firmware_extractor/tools/Linux/bin/packsparseimg:
  -o string
        output dir path (default "./")
  -t string
        image to unsparse: system / cache / userdata / persist / vendor (default "system")
  -x string
        xml file to load (default "./rawprogram_unsparse.xml")
mv: cannot stat 'dtb.raw': No such file or directory
flag needs an argument: -x
Usage of /home/yuifunami/dumpyara/Firmware_extractor/tools/Linux/bin/packsparseimg:
  -o string
        output dir path (default "./")
  -t string
        image to unsparse: system / cache / userdata / persist / vendor (default "system")
  -x string
        xml file to load (default "./rawprogram_unsparse.xml")
mv: cannot stat 'boot.raw': No such file or directory
flag needs an argument: -x
Usage of /home/yuifunami/dumpyara/Firmware_extractor/tools/Linux/bin/packsparseimg:
  -o string
        output dir path (default "./")
  -t string
        image to unsparse: system / cache / userdata / persist / vendor (default "system")
  -x string
        xml file to load (default "./rawprogram_unsparse.xml")
mv: cannot stat 'tz.raw': No such file or directory
```
Firmware dumped myself using EDL tool, it fails to process it properly. It does have all partitions needed in raw image .bin files.

**Comment by Hadenix** [comment]
I have the same problem

**Comment by ant14181** [comment]
> I have the same problem

This should've been fixed by fb2cdb68b252e16e7a983d45f4b22b6a1e3362f4 (_fix(QFIL): resolve issues with recent packages_), but doesn't seem the case. 

Could you provide more information, like a link to the package you tried to extract and more? The former would be estremely helpful especially because I can debug this occurrence by myself.

Let me know as soon as possible.

---
Related might be an issue regarding a typo I've done on a variable, check a1e40d2b94e39d929d5bb3aa1ac04f1c16b5674b (_fix(73ab06f01a78288aea32a658d518ca23866a46d8): resolve typo on 'RAWPROGRAM'_). I've only noticed this now and not later. Hope this was the culprit.
